lua/telescope/pickers/highlights.lua

```diff
@@ -56,65 +61,106 @@ end
 
 function Highlighter:hi_sorter(row, prompt, display)
   local picker = self.picker
+  local sorter = picker.sorter
```

Comment:
there is no need for having `sorter` in a local var. Its only used in line 74 so we can just do `picker.sorter` there. esp because line 65 also uses picker.sorter